Fails to build on 32-bit architectures

Regressed by 65d96bf6c158. See error logs: [i386](http://beefy1.nyi.freebsd.org/data/113i386-quarterly/532056/logs/errors/citra-qt5-s20200418.log), [armv7](https://github.com/citra-emu/citra/files/4498202/citra-s20200418.log).
```c++
In file included from src/input_common/udp/udp.cpp:10:
In file included from src/./core/settings.h:13:
In file included from src/./core/hle/service/cam/cam.h:21:
In file included from src/./core/hle/service/service.h:17:
In file included from src/./core/hle/kernel/hle_ipc.h:21:
In file included from src/./core/hle/ipc.h:9:
In file included from src/./core/hle/kernel/thread.h:18:
In file included from src/./core/arm/arm_interface.h:16:
src/./core/memory.h:110:15: error: class member cannot be redeclared
        Entry operator[](std::size_t idx) {
              ^
src/./core/memory.h:102:15: note: previous definition is here
        Entry operator[](VAddr idx) {
              ^
```

--- a/src/core/memory.h
+++ b/src/core/memory.h
@@ -99,14 +99,6 @@ struct PageTable {
             VAddr idx;
         };
 
-        Entry operator[](VAddr idx) {
-            return Entry(*this, idx);
-        }
-
-        u8* operator[](VAddr idx) const {
-            return raw[idx];
-        }
-
         Entry operator[](std::size_t idx) {
             return Entry(*this, static_cast<VAddr>(idx));
         }

